raiden/api/rest.py

```diff
@@ -525,6 +528,39 @@ def get_our_address(self) -> Response:
     def get_raiden_version(cls) -> Response:
         return api_response(result=dict(version=get_system_spec()["raiden"]))
 
+    def get_contract_versions(self) -> Response:
+        contracts = dict(
+            contracts_version=CONTRACTS_VERSION,
+            token_network_registry_address=to_checksum_address(
+                self.raiden_api.raiden.default_registry.address
+            ),
+            secret_registry_address=to_checksum_address(
+                self.raiden_api.raiden.default_secret_registry.address
+            ),
+            service_registry_address="",
+            user_deposit_address="",
+            monitoring_service_address="",
+            one_to_n_address="",
+        )
+        if self.raiden_api.raiden.default_service_registry is not None:
+            contracts["service_registry_address"] = to_checksum_address(
+                self.raiden_api.raiden.default_service_registry.address
+            )
+        if self.raiden_api.raiden.default_user_deposit is not None:
+            contracts["user_deposit_address"] = to_checksum_address(
+                self.raiden_api.raiden.default_user_deposit.address
+            )
+        if self.raiden_api.raiden.default_msc_address is not None:
+            contracts["monitoring_service_address"] = to_checksum_address(
+                self.raiden_api.raiden.default_msc_address
+            )
+        if self.raiden_api.raiden.default_one_to_n_address is not None:
+            contracts["one_to_n_address"] = to_checksum_address(
+                self.raiden_api.raiden.default_one_to_n_address
+            )
+
+        return api_response(result=contracts)
```

Comment:
@manuelwedler Any other information you need?

Reply:
Looks perfect. Thank you!